services/common/src/redux/createAppSlice.ts

```diff
@@ -0,0 +1,6 @@
+import { buildCreateSlice, asyncThunkCreator } from "@reduxjs/toolkit";
+
+// https://redux-toolkit.js.org/api/createslice#createasyncthunk
+export const createAppSlice = buildCreateSlice({
```

Comment:
literally copied and pasted this from their documentation. For bundle size reasons.